File: src/refactorings/replace-control-flag-with-break/fixtures/nested-loop.fixture.py
params = {"file": "fixture.py", "target": "done"}

def find_in_matrix(matrix, target):
    done = False
    result = (-1, -1)
    for i, row in enumerate(matrix):
        for j, val in enumerate(row):
            if val == target:
                result = (i, j)
                done = True
        if done:
            break
    return result

def main():
    matrix = [[1, 2], [3, 4], [5, 6]]
    result = find_in_matrix(matrix, 4)
    return str(result)

File: src/refactorings/replace-loop-with-pipeline/fixtures/nested-loop.fixture.ts
export const params = { file: "fixture.ts", target: "7" };

export function main(): string {
  const matrix = [[1, 2], [3, 4]];
  const processed: string[] = [];
  for (const row of matrix) {
    for (const cell of row) {
      const doubled = cell * 2;
      processed.push(String(doubled));
    }
  }
  return processed.join(",");
}
